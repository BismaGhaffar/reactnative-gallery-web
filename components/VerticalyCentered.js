import styled from 'styled-components'

const VerticalyCentered = styled.div`
  margin-top: 50vh;
  transform: translateY(-50%);
`

export default VerticalyCentered
