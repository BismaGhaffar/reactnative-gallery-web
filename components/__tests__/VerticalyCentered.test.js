import React from 'react'
import renderer from 'react-test-renderer'
import VerticalyCentered from '../VerticalyCentered'

it('VerticalyCentered can be created', () => {
  const comp = renderer.create(<VerticalyCentered />)
  expect(comp).toBeDefined()
})

it('<VerticalyCentered /> toMatchSnapshot', () => {
  const tree = renderer.create(<VerticalyCentered />).toJSON()
  expect(tree).toMatchSnapshot()
})
