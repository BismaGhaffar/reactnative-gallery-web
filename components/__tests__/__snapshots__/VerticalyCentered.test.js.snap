// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VerticalyCentered /> toMatchSnapshot 1`] = `
<div
  className="VerticalyCentered-s2kwuh3-0 fEVrzi"
/>
`;
